<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>382</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>25</width>
    <height>25</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ABC IO</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QScrollArea" name="scrollArea_all">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>451</width>
         <height>317</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lin_export_path">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none</string>
            </property>
            <property name="inputMask">
             <string/>
            </property>
            <property name="text">
             <string>D:/Repo</string>
            </property>
            <property name="dragEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_sel_path">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>1</width>
              <height>1</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icon/file.png</normaloff>icon/file.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;导出选项：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="radio_cf">
            <property name="text">
             <string>Current frame</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radio_ts">
            <property name="text">
             <string>Time Slider</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radio_se">
            <property name="text">
             <string>Start/End</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Start/End&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QCheckBox" name="check_vis_path">
          <property name="text">
           <string>自动匹配相机路径</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QDoubleSpinBox" name="float_start">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>-10000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10000.000000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="float_end">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>-10000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10000.000000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="9" column="1">
         <widget class="QDoubleSpinBox" name="float_step">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;输出路径：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="15" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>1000</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="11" column="1">
         <widget class="QCheckBox" name="check_cam">
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;导出相机是 FBX 格式&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>相机</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Step：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QCheckBox" name="check_body">
            <property name="text">
             <string>人物</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="check_prop">
            <property name="text">
             <string>道具</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="check_BG">
            <property name="text">
             <string>BG</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="check_color_set">
            <property name="text">
             <string>面部颜色集</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="check_export_sel">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>选择导出</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_cjjc">
        <property name="text">
         <string>场景检测</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_export">
        <property name="text">
         <string>Export</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>check_vis_path</sender>
   <signal>toggled(bool)</signal>
   <receiver>lin_export_path</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_vis_path</sender>
   <signal>toggled(bool)</signal>
   <receiver>btn_sel_path</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_se</sender>
   <signal>toggled(bool)</signal>
   <receiver>float_start</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_se</sender>
   <signal>toggled(bool)</signal>
   <receiver>float_end</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>563</x>
     <y>500</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_export_sel</sender>
   <signal>toggled(bool)</signal>
   <receiver>check_body</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_export_sel</sender>
   <signal>toggled(bool)</signal>
   <receiver>check_color_set</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_export_sel</sender>
   <signal>toggled(bool)</signal>
   <receiver>check_BG</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_export_sel</sender>
   <signal>toggled(bool)</signal>
   <receiver>check_prop</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_export_sel</sender>
   <signal>toggled(bool)</signal>
   <receiver>check_cam</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
